src/search/dag_classic/wrapper.cc

```diff
@@ -0,0 +1,144 @@
+/*
+  This file is part of Leela Chess Zero.
+  Copyright (C) 2025 The LCZero Authors
+
+  Leela Chess is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Leela Chess is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
+
+  Additional permission under GNU GPL version 3 section 7
+
+  If you modify this Program, or any covered work, by linking or
+  combining it with NVIDIA Corporation's libraries from the NVIDIA CUDA
+  Toolkit and the NVIDIA CUDA Deep Neural Network library (or a
+  modified version of those libraries), containing parts covered by the
+  terms of the respective license agreement, the licensors of this
+  Program grant you additional permission to convey the resulting work.
+*/
+
+#include "chess/gamestate.h"
+#include "search/dag_classic/search.h"
+#include "search/dag_classic/stoppers/factory.h"
+#include "search/register.h"
+#include "search/search.h"
+
+namespace lczero {
+namespace dag_classic {
+namespace {
+
+const OptionId kThreadsOptionId{
+    "threads", "Threads",
+    "Number of (CPU) worker threads to use, 0 for the backend default.", 't'};
+
+const OptionId kClearTree{"", "ClearTree",
+                          "Clear the tree before the next search."};
+
+class DagClassicSearch : public SearchBase {
+ public:
+  DagClassicSearch(UciResponder* responder, const OptionsDict* options)
+      : SearchBase(responder), options_(options) {}
+
+ private:
+  void NewGame() override;
+  void SetPosition(const GameState& pos) override;
+  void StartSearch(const GoParams&) override;
+  void StartClock() override {
+    move_start_time_ = std::chrono::steady_clock::now();
+  }
+  void WaitSearch() override {
+    if (search_) search_->Wait();
+  }
+  void StopSearch() override {
+    if (search_) search_->Stop();
+  }
+  void AbortSearch() override {
+    if (search_) search_->Abort();
+  }
+
+  const OptionsDict* options_;
+  std::unique_ptr<dag_classic::TimeManager> time_manager_;
```

Comment:
Hmm as the `wrapper.cc` is already in the namespace `dag_classic`, there's no need to explicitly mention the namespace everywhere. I'll do the same change for `classic` `wrapper.cc`